France-ioi/AlgoreaFrontend
# Feature/use p button instead of alg button

## Description

Fixes #961

## Notes (out of scope, known isues, hints for reviewing code, ...)  (optional)

...

## Test cases

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/use-p-button-instead-of-alg-button/en/#/activities/by-id/1980584647557587953;path=4702,4102;attempId=0/edit)
  3. Change description field
  4. Then I see "Save" and "Cancel Changes" buttons in bottom

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/use-p-button-instead-of-alg-button/en/#/groups/by-id/50118905546577977;path=/details/members)
  3. Scroll down to "Invite Users" section
  4. Then I see "Invite" button

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/use-p-button-instead-of-alg-button/en/#/groups/by-id/672913018859223173;path=52767158366271444/details)
  3. Scroll down to "Leave the group?" section
  4. Then I see "Leave This Group" button

  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/use-p-button-instead-of-alg-button/en/#/groups/mine)
  3. Scroll down to "Join a group by code" section
  4. Then I see "Join" button with icon

- [x] Case 5:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/use-p-button-instead-of-alg-button/en/#/groups/by-id/4035378957038759250;path=/details/managers)
  3. Scroll down to "Add a manager" section
  4. Then I see "Add" button with icon

- [x] Case 6:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/use-p-button-instead-of-alg-button/en/#/groups/by-id/672913018859223173;path=52767158366271444/details)
  3. Then I start watch group
  4. Then I click on "test 2 (Pixal)" of suggestions
  5. And I click on "«Pixal» can access this activity" header
  6. Then I click on "Edit Permissions" button
  7.  Then I see "Cancel" and "Proceed" buttons with icons

- [x] Case 7:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/use-p-button-instead-of-alg-button/en/#/groups/by-id/672913018859223173;path=52767158366271444/details/managers)
  3. Then I click on edit on one of the manager in managers list
  4.  Then I see "Cancel" and "Save" buttons with icons

- [x] Case 8:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/use-p-button-instead-of-alg-button/en/#/activities/by-id/1980584647557587953;path=4702,4102;attempId=0/edit)
  3. Then I scroll down to the "Add a content" section
  4. Then I search by word "Additions"
  5. Then I see "Add" button with icon

- [x] Case 9:
  1. Given I am the usual user
  2. When I go to [this page](https://dev.algorea.org/branch/feature/use-p-button-instead-of-alg-button/en/#/groups/by-id/2116767266653043949;path=/details/members)
  3. Then I scroll down to the "Let user join using a code" section
  4. Then I see "Generate a code" button with icon
